yarn-project/types/src/tx_execution_request.ts

```diff
@@ -17,19 +18,23 @@ export class TxExecutionRequest {
     /**
      * Function arguments.
      */
-    public args: Fr[],
+    public argsHash: Fr,
```

Comment:
Does it make sense to keep this field as just `args`, and have the `packedArgs` field as optional, just to carry extra args hashes that may be needed for nested calls?

Reply:
I felt that this way was more coherent with the public inputs that also carry the args hash, etc 🤔